applications/mp4box/main.c

```diff
@@ -1850,6 +1852,12 @@ const char *grab_m2ts = NULL;
 const char *grab_ifce = NULL;
 #endif
 FILE *logfile = NULL;
+Bool mpu_is_complete=0;
```

Comment:
Is there another way to retrieve the parameters found in mp4box_parse_args function ?